lib/Twig/Extensions/Template/DisplayCallGraphTemplate.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) 2010 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Displays the call graph (templates names and block names) in a visual way directly in the generated HTML.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+abstract class Twig_Extensions_Template_DisplayCallGraphTemplate extends Twig_Template
+{
+    /**
+     * @var string[]
+     */
+    protected $templateBlackList = array();
```

Comment:
By inheritance but I'll move that to a getter like for other properties, thanks.